### PR TITLE
Add integration test using Route 53 API

### PR DESCRIPTION
**Pull request by domcleal**

### CODE REVIEW

File: test/dns_route53_integration_test.rb [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+
+  def test_delete_aaaa_record
+    create_record forward_zone, "test.#{forward_zone}.", 'AAAA', '3600', ['2001:db8::1']
+    delete "/test.#{forward_zone}/AAAA"
+    assert_equal 200, last_response.status
+    assert_nil record(forward_zone, "test.#{forward_zone}.", 'AAAA')
+  end
+
+  def test_delete_explicit_cname_record
+    create_record forward_zone, "test.#{forward_zone}.", 'CNAME', '3600', ['test.example.com']
```
Comment (ekohl) [inline test/dns_route53_integration_test.rb:101]: Doesn't this need a trailing dot as well?

**Comment by ekohl** [comment]
It looks like you're running against a subdomain of theforeman.org. I hope that the account has no access to theforeman.org so it can't accidentally remove everything of theforeman.org.

File: test/dns_route53_integration_test.rb (hunk 1 above)
Comment (domcleal) [reply test/dns_route53_integration_test.rb:101]: Thanks, fixed.

**Comment by domcleal** [comment]
Yes, it's just a separate zone and IAM user in my own account - unrelated to whatever theforeman.org's DNS is. I'm leaving it disabled on Travis CI as the throttling is preventing it from running reliably.

**Pull request closed without merging**